### PR TITLE
Capstone Project PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

I created an Anime and Comicon Fan Expo for my capstone project.

In this project, I created two HTML files, one for the homepage and one for the about page. The pages are also responsive and in the mobile version, the user can use the hamburger for the mobile menu. 

Looking forward to hearing your ideas.

Thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      </div>
+    </section>
+    <footer class="footer-desktop">
+      <div id="footer-logo">
+        <img src="./Images/Logo.png" alt="logo" id="footer-img" />
+      </div>
+      <div id="footer-info">
+        <p>Lorem ipsum dolor sit amet consectetur</p>
+      </div>
+    </footer>
```
Comment (Qoosim) [inline index.html]: - [x] It is required to stick to the guiding design as close as possible. This includes, font-type, font-size, font-family, color, background-color, margin and padding among others. The margin and padding of your partner and footer sections are not in line with what we have on the guiding template. Kindly give a closer look at the guiding design and walk your design alongside.
- [x] Also, there is horizontal scroll at the bottom of your page. Kindly correct that too.

### Your Design (Desktop)
your-home-footer-spacing

### Template (Desktop)
temp-home-footer-spacing

File: about.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      </div>
+    </section>
+    <footer class="footer-mobile">
+      <div id="footer-logo">
+        <img src="./Images/Logo.png" alt="logo" id="footer-img" />
+      </div>
+      <div id="footer-info">
+        <p>Lorem ipsum dolor sit amet consectetur</p>
+      </div>
+    </footer>
```
Comment (Qoosim) [inline about.html]: - [x] On the guiding design, the background color is black, but your design carries something different. Kindly look into that and fix.
- [x] You should also keep in mind the spacing of your footer compare to the guiding template.

### Your Design (Desktop)
your-about-footer-desktop-bg

### Guiding Template (Desktop)
about-footer-desktop-bg

File: about.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      </nav>
+    </header>
+    <header class="mobile-menu">
+      <div class="close-icon"><i class="fa-solid fa-xmark"></i></div>
+      <div href="./index.html" class="menu-links">Home</div>
+      <div href=".main-program" class="menu-links">Program</div>
+      <div href="#join-link" class="menu-links">Join</div>
+      <div href="./about.html/#partners-mobile" class="menu-links">Sponsor</div>
+      <div class="menu-links">News</div>
+    </header>
```
Comment (Qoosim) [inline about.html]: - [x] From about page on the mobile version. When clicking on the hamburger, the mobile menu would appear. It is expected to redirect a user from about page to home page when `home` menu item is clicked. But the behaviour in your case is not. It remains on the about page despite clicking on the home menu item number of times. Kindly look into that and fix. 

[mobile-menu-about.webm](https://user-images.githubusercontent.com/26597432/212057082-614db5da-4644-45e4-bf41-d76399b53a8f.webm)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+      </nav>
+    </header>
+    <header class="mobile-menu">
+      <div class="close-icon"><i class="fa-solid fa-xmark"></i></div>
+      <div href="./about.html" class="menu-links">About</div>
+      <div href=".main-program" class="menu-links">Program</div>
+      <div href="#join-link" class="menu-links">Join</div>
+      <div href="./about.html/#partners-mobile" class="menu-links">Sponsor</div>
+      <div class="menu-links">News</div>
+    </header>
```
Comment (Qoosim) [inline index.html]: - [x] From home page on the mobile version. When clicking on the hamburger, the mobile menu would appear. It is expected to redirect a user from home page to about page when `about` menu item is clicked. But the behaviour in your case is not. It remains on the home page despite clicking on the about menu item number of times. Kindly look into that and fix. 
[mobile-menu-home.webm](https://user-images.githubusercontent.com/26597432/212057560-ebf7fce2-0ef3-41cc-9d04-792172a880b8.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @mertkantaroglu, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Top Highlights
-  No linter errors. :heavy_check_mark: 
-  You wrote a good README file, but not professional. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-   Descriptive PR title and summary. :heavy_check_mark: 




## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

- [ ] Good jobs for making your project to this level. You've actually put in a lot of works in there. You're expected to make a video presentation of your capstone project, not your portfolio project. Kindly redo a new presentation video that walk us through the work you've done in the project and show us pieces of code you're most proud of.

**Comment by mertkantaroglu** [comment]
Hello,

Although I fixed the issues mentioned in the previous PR, the deployed page shows the previous version. Therefore, please run the code from the live version to see the corrections.

Thanks.

**Review by alexander16108 — APPROVED** [review]
Hi @mertkantaroglu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- No linter errors. ✔️
- You wrote a good README file, but not professional. ✔️
- Correct GitHub Flow. ✔️
- Descriptive PR title and summary. ✔️

## Optional suggestions
- [ ] you can add eiichiro oda the author of one piece as a speaker 😄
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**